sdk/appcenter/src/main/java/com/microsoft/appcenter/http/HttpClientRetryer.java

```diff
@@ -44,13 +46,15 @@
      */
     private final Random mRandom = new Random();
 
+    private final boolean mRetryOnNetworkingFailures;
+
     /**
      * Init with default retry policy.
      *
      * @param decoratedApi API to decorate.
      */
-    HttpClientRetryer(HttpClient decoratedApi) {
-        this(decoratedApi, new Handler(Looper.getMainLooper()));
+    HttpClientRetryer(boolean retryOnNetworkingFailures, HttpClient decoratedApi) {
```

Comment:
But now if our own endpoint is down, this will still hang for a long time.
Do we really need to retry anything in Data if we want to fail fast and go offline mode?